### ISSUE TITLE
Site Info on Footer appear double

### ISSUE THREAD

**Issue by rinazrina** [woocommerce/storefront#2091]
## Describe the bug
The site info on Footer appears twice on sites using Storefront and WooCommerce. 

Isolating the problem (mark completed items with an [x]):
- [x] I have deactivated other plugins and themes and confirmed this bug occurs when only WooCommerce + Storefront theme are active.
- [x] I can reproduce this bug consistently using the steps below.

## To Reproduce

1. Activate Storefront theme and WooCommerce plugin
2. Leave all other plugins inactive
3. Check the default site footer that says: © Site title [Built with Storefront & WordPress.com](https://wordpress.com/?ref=footer_website). It will appear twice
4. Try to deactivate WooCommerce plugin and the text will appear once as it should

## Screenshots
<img width="811" alt="Markup 2023-02-16 at 23 26 27" src="https://user-images.githubusercontent.com/33721365/219410701-722c2dc6-797b-406e-a149-efb0bea449ee.png">

## Expected behavior
The site info on footer does not appear double. 

### Browser Environment

- Platform: macOS
- Browser(s): Chrome

### WordPress Environment
Storefront version 4.2.0
WooCommerce version 7.4.0

<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://ecommercerinatest12345.wpcomstaging.com
Site address (URL): https://ecommercerinatest12345.wpcomstaging.com
WC Version: 7.4.0
REST API Version: ✔ 7.4.0
WC Blocks Version: ✔ 9.4.3
Action Scheduler Version: ✔ 3.5.4
Log Directory Writable: ✔
WP Version: 6.1.1
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 7.4.33
PHP Post Max Size: 2 GB
PHP Time Limit: 1200
PHP Max Input Vars: 6144
cURL Version: 7.86.0
OpenSSL/1.1.1n

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.4.26-MariaDB-log
Max Upload Size: 2 GB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 7.4.0
WC Database Prefix: wp_
Total Database Size: 6.39MB
Database Data Size: 4.56MB
Database Index Size: 1.83MB
wp_woocommerce_sessions: Data: 0.19MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.09MB + Index: 0.13MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.11MB + Engine InnoDB
wp_e_events: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_failed_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_options: Data: 2.19MB + Index: 0.08MB + Engine InnoDB
wp_postmeta: Data: 0.52MB + Index: 0.09MB + Engine InnoDB
wp_posts: Data: 0.36MB + Index: 0.06MB + Engine InnoDB
wp_queue: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.03MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yoast_indexable: Data: 0.08MB + Index: 0.09MB + Engine InnoDB
wp_yoast_indexable_hierarchy: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yoast_primary_term: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_yoast_seo_links: Data: 0.02MB + Index: 0.03MB + Engine InnoDB

### Post Type Counts ###

amp_validated_url: 1
attachment: 59
customize_changeset: 9
custom_css: 2
elementor_library: 1
feedback: 2
jetpack_migration: 2
jp_img_sitemap: 1
jp_sitemap: 1
jp_sitemap_master: 1
jp_vid_sitemap: 1
nav_menu_item: 21
page: 20
post: 28
product: 8
product_variation: 3
revision: 63
wc_membership_plan: 1
wc_user_membership: 1
wpcode: 2
wp_global_styles: 16
wp_navigation: 2
wp_template_part: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (3) ###

Akismet Anti-Spam: by Automattic – 5.0.2
Jetpack: by Automattic – 11.9-a.1
WooCommerce: by Automattic – 7.4.0

### Inactive Plugins (43) ###

AMP: by AMP Project Contributors – 2.4.0
Blockify: by Blockify – 0.6.4
Classic Editor: by WordPress Contributors – 1.6.2
CoBlocks: by GoDaddy – 2.25.5
Crowdsignal Forms: by Automattic – 1.6.6
Crowdsignal Polls & Ratings: by Automattic
Inc. – 3.0.10

Easy Google Fonts: by Titanium Themes – 2.0.4
Elementor: by Elementor.com – 3.11.1
Facebook for WooCommerce: by Facebook – 3.0.11
Gutenberg: by Gutenberg Team – 15.0.1
Health Check & Troubleshooting: by The WordPress.org community – 1.5.1
Instant Indexing: by Rank Math – 1.1.16
Jotform oEmbed: by Jotform.com – 1.3.0
Klarna Checkout for WooCommerce: by Krokedil – 2.10.0
Klarna Payments for WooCommerce: by krokedil
klarna
automattic – 2.12.1

Layout Grid: by Automattic – 1.8.2
Lord of the Files: Enhanced Upload Security: by Blobfolio
LLC – 1.3.12

Mailchimp for WooCommerce: by Mailchimp – 2.7.6
Page Optimize: by Automattic – 0.5.2
Sensei LMS: by Automattic – 4.11.1
Storefront Powerpack: by WooCommerce – 2.0.0
TaxJar - Sales Tax Automation for WooCommerce: by TaxJar – 4.1.5
WooCommerce Admin: by WooCommerce – 3.3.2
WooCommerce Australia Post Shipping: by WooCommerce – 2.5.0
WooCommerce Canada Post Shipping: by WooCommerce – 2.7.1
WooCommerce Direct Checkout: by QuadLayers – 2.7.7
WooCommerce Eway Payment Gateway: by WooCommerce – 3.5.2
WooCommerce Memberships: by SkyVerge – 1.17.1
WooCommerce PayFast Gateway: by WooCommerce – 1.5.0
WooCommerce PayPal Checkout Gateway: by WooCommerce – 2.1.3
WooCommerce Product Add-ons: by WooCommerce – 6.0.3
WooCommerce Royal Mail: by WooCommerce – 2.7.1
WooCommerce Shipping & Tax: by WooCommerce – 2.2.3
WooCommerce Square: by WooCommerce – 3.5.0
WooCommerce Stripe Gateway: by WooCommerce – 7.1.0
WooCommerce UPS Shipping: by WooCommerce – 3.4.1
WooCommerce USPS Shipping: by WooCommerce – 4.6.2
WordPress.com Editing Toolkit: by Automattic – 3.56293
WordPress Importer: by wordpressdotorg – 0.8
WP Add Mime Types: by Kimiya Kitani – 3.0.3
WPCode Lite: by WPCode – 2.0.7
WP Extra File Types: by Davide Airaghi – 0.5.2
Yoast SEO: by Team Yoast – 20.1

### Dropin Plugins (2) ###

advanced-cache.php: advanced-cache.php
object-cache.php: Memcached

### Must Use Plugins (2) ###

Health Check Troubleshooting Mode: by  – 1.8.1
WP.com Site Helper: by  –

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔
Enforce Approved Product Download Directories: –
Order datastore: WC_Order_Data_Store_CPT

### WC Pages ###

Shop base: #406 - /shop/
Cart: #343 - /cart/
Checkout: #345 - /checkout/
My account: #347 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 4.2.0
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
multichannel-marketing
mobile-app-banner
new-product-management-experience
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: minified-js
navigation
product-variation-management
settings

Daily Cron: ✔ Next scheduled: 2023-02-17 06:43:04 +00:00
Options: ✔
Notes: 103
Onboarding: skipped

### Action Scheduler ###

Complete: 177
Oldest: 2023-01-17 09:37:29 +0000
Newest: 2023-02-16 15:33:21 +0000

Failed: 10
Oldest: 2023-02-16 01:33:50 +0000
Newest: 2023-02-16 15:24:11 +0000

Pending: 2
Oldest: 2023-02-16 16:24:11 +0000
Newest: 2023-02-17 01:33:50 +0000


### Status report information ###

Generated at: 2023-02-16 15:34:11 +00:00
`
```
</details>

**Comment by Aljullu** [comment]
Thanks for opening this issue, @rinazrina!

Unfortunately, I'm not able to reproduce on my end. Can you verify you don't have any custom code interfering with the way Storefront renders the footer? I would suggest testing in a new WordPress fresh installation to make sure there are no plugins or custom code snippets that might be interfering.

It would also be helpful if you can tell us whether that's an issue that appeared after an update or you were experimenting it for a long time.

Thanks in advance!

**Comment by rinazrina** [comment]
Thank you for looking @Aljullu 🙏🏼

> Can you verify you don't have any custom code interfering with the way Storefront renders the footer?

Yes, I don't have any custom codes on my test site. 

> It would also be helpful if you can tell us whether that's an issue that appeared after an update or you were experimenting it for a long time.

I'm not sure when it happened exactly. I just noticed this now when I got a report from a customer. 

I've got a couple of similar reports from other users who are using Storefront child themes, one of them is Homestore. 

Upon further checking, the issue seems to only happens when the site has a WP.com eCommerce plan, and not on WP.com Business plan. I've tested this using three of my test sites. 

I hope that helps narrow down the issue.

**Comment by WillBrubaker** [comment]
5974789-zen

**Comment by kaushikasomaiya** [comment]
5969956-zen

**Comment by albarin** [comment]
Hey, I've been able to reproduce this issue using the https://github.com/Automattic/wc-calypso-bridge plugin locally.
The issue was recently introduced by this commit some days ago https://github.com/Automattic/wc-calypso-bridge/commit/410a81d48e043b0d2f8dc23519aa98a7845f0525#diff-574140c5ebef5f5a93893e45669a487fc8f736639e6eb3b19f0869e1735c95beR44-R45
I think the problem is the `remove_action` is not working because the filter is not yet there when trying to remove it, then the `add_action` adds another footer and we end up with two.
Maybe @xristos3490 can also have a look and help here, thanks! 🙏

**Comment by xristos3490** [comment]
> I think the problem is the remove_action is not working because the filter is not yet there when trying to remove it, then the add_action adds another footer and we end up with two.

Hey @albarin! Great investigation skills! That is exactly the issue here. 

I've opened a PR in The Bridge to resolve this here: https://github.com/Automattic/wc-calypso-bridge/pull/970 Will release it ASAP.

**Comment by albarin** [comment]
Great! Thanks a lot @xristos3490 🙌

**Comment by xristos3490** [comment]
The fix is included in the WC Calypso Bridge v2.0.2 and should be deployed in all sites by EOD.
